.github/workflows/deploy-grafana-dashboards.yaml

```diff
@@ -24,13 +24,17 @@ jobs:
           python3 -m pip install -r requirements.txt
           sudo apt install jsonnet
 
+      - name: Authenticate to Google Cloud
+        uses: google-github-actions/auth@v0.6.0
+        with:
+          credentials_json: ${{ secrets.GCP_KMS_DECRYPTOR_KEY }}
```

Comment:
Perhaps you need `project_id` set here?

This is a copy-paste from a github workflow that successfully prepares for sops to be used.

```yaml
      - uses: google-github-actions/auth@v0
        with:
          project_id: ${{ secrets.GCP_PROJECT_ID }}
          credentials_json: ${{ secrets.GCP_CREDENTIALS }}
```